src/components/NgoDetailsDirection/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import GoogleMapReact from "google-map-react";
+import LocationPin from "../LocationPin";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const NgoDetailsDirection = ({ address }) => {
```

Comment:
what is the usage of this props in the component, I dont see it getting used

Reply:
I use it in location  const locationNgoInMaps = {
    address, // this is where I use=> address: address => so I did just address 
    lat: 37.42216,
    lng: -122.08427,
  };

Reply:
@fadime-ozdemir yes but the props are not getting used, you are using the variable u have directly

Reply:
@fadime-ozdemir oh okai just saw that